pkgs/development/libraries/science/math/faiss/default.nix

```diff
@@ -48,11 +48,16 @@ stdenv.mkDerivation {
 
   src = fetchFromGitHub {
     owner = "facebookresearch";
-    repo = pname;
-    rev = "v${version}";
+    repo = "faiss";
+    rev = "refs/tags/v${version}";
     hash = "sha256-P8TynU6jz5NbcWLdI7n4LX5Gdz0Ks72bmOzQ3LGjQCQ=";
   };
 
+  postPatch = lib.optionalString pythonSupport ''
+    substituteInPlace faiss/python/loader.py \
+      --replace-fail "import numpy.distutils.cpuinfo" "return False"
```

Comment:
THere are two matches in `loader.py`, and the 2nd one in a function that, I think, is meant to return a dict so this doesn't type check. Maybe we replace `# platform-dependent legacy fallback using numpy.distutils.cpuinfo` with the return statement? xD

Reply:
Sure, I had observed this, but didn't think that it was too much of an issue.
Your idea is indeed better.
I pushed it.